### ISSUE TITLE
[en] Markup Languages [Page 18]

### ISSUE THREAD

**Issue by gilaro** [contextgarden/not-so-short-introduction-to-context#16]
There are conceptual errors in the topic about markup languages.

The following distinction seems irrelevant to me:

- no _typographical_ markup language
- no _conceptual_ markup language

Markups are a way to transmit informations but the correlation is mainly at the operating level.

- markups
- commands
- macros
- ...

(Lua)TeX uses _control sequences_ while HTML (Hyper Text) and XML (eXtended) may be used by several softwares. 

In summary, you are confusing several things.
